Documentation/SpatialAwareness/SpatialObjectMeshObserver.md

```diff
@@ -1,78 +1,78 @@
-# The Spatial Object Mesh Observer
+# Configuring Mesh Observers for Editor
 
-The spatial object mesh observer is an editor-only data provider for the Spatial Awareness system 
-that enables using an imported 3D model to represent the spatial mesh. One common use of the 
-spatial object mesh observer is to use data scanned via a Microsoft HoloLens to test how an
```

Comment:
Why is it called an _observer_? I don't really understand the naming.

Reply:
They are data providers that surface mesh data by observing the environment for a specific platform. 

I tried better spelling that out in the Getting Started page

Reply:
> Why is it called an observer?

In large part the name was chosen to match the other providers. :)